### PR TITLE
Milestone 3 : Created final mobile

### PR DESCRIPTION
**Pull request by Theodoraldo**
## Portfolio : mobile version

-Add work section of built projects
-Add about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```
Comment (Shisui6) [inline index.html:1]: You've done well creating the works and about sections but please note that you are not supposed to create a new repository for every new milestone😅. You are supposed to use the same repository and create  new branches for each milestone to be merged into the main branch. This way, you build out your portfolio in one repository step by step. Please add your approved code for the toolbar and headline sections into this repository and so your portfolio will have all sections🙏.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="custom.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+    />
+    <title>Portfolio-Mobile Fiirst</title>
+  </head>
+  <body>
+    <div class="container">
```
Comment (Shisui6) [inline index.html]: Kindly wrap your works section in a `<section>` tag🙏. This is for semantic reasons, as section tags are used to identify different parts of a webpage

File: custom.css [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  gap: 88px;
+}
+
+.card-body {
+  border-radius: 16px;
+  padding: 16px;
+  background-color: #fff;
+  height: 100%;
+  width: 327px;
+}
```
Comment (Shisui6) [inline custom.css]: Kindly update the works section to match the figma design better🙏

Your design:
image

Figma design:
image

A few things to note:
- Please make the left and right margins of each card even
- Please update to background color of the section to match the template. I believe `#f7f7f9` should do it

File: index.html [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="show">See Project</button>
+        </div>
+      </div>
+    </div>
+
+    <section class="banner">
+      <h1 class="banner-title">About Myself</h1>
```
Comment (Shisui6) [inline index.html]: Kindly update this element to use another `<hx>` element apart from `<h1>`🙏. This is for semantic reasons, as a good webpage is typically supposed to have one `<h1>` tag which represents the main title

File: index.html [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+            <path
+              d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z"
+              fill="#505F79"
+            />
+          </svg>
+        </div>
+        <button class="show">Get my resume</button>
+
+        <ul class="about-cat">
+          <li>Languages</li>
```
Comment (Shisui6) [inline index.html:528]: Your about section is looking really good✨. But please add the arrow icons next to each technology category as shown in the figma design.

Your design:
image

Figma design:
image

File: custom.css [hunk 6]
```diff
@@ -0,0 +1,181 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap");
```
Comment (Shisui6) [inline custom.css:1]: Please remove all lines that start with `>` from your readme file🙏. These lines are comments meant to help you the project owner fill out each section properly and is not meant for your users to see. It makes your readme look unprofessional

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Theodoraldo, 

# Requires Changes 🍀
You have done a great job so far!
Your design is good 🖼️, and your code follows best practices. However, there are a few issues that you still need to work on🪲.

## Highlights
- Good design ✅
- Nice use of grid and flexbox ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,707 @@
...
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="show">See project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="banner">
+      <h1 class="banner-title">About Myself</h1>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Kindly update this element to use another `hx` element apart from `h1`🙏. This is for semantic reasons, as a good webpage is typically supposed to have one h1 tag which represents the main title.
- This was highlighted by the previous reviewer, kindly make the change

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Theodoraldo      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights:
✔️ README setup
✔️  Linters passed
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes needed has to do with semantics

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ouasamine — APPROVED** [review]
Hi @Theodoraldo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Nice work on the UI ✔️ 
- Good work on the project's documentation 👏🏽 👏🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**